CMakeLists.txt

```diff
@@ -302,6 +302,13 @@ ENDIF()
 
 IF (ASPECT_PRECOMPILE_HEADERS)
   MESSAGE(STATUS "Enabling cotire to precompile external header files.")
+
+  # Request new BYPRODUCTS behavior
+  CMAKE_MINIMUM_REQUIRED(VERSION 3.2.0)
```

Comment:
I checked this and I was going around this in what was likely the incorrect manner.